posts/2022-07-05-logrecordcontext-22007.adoc

```diff
@@ -0,0 +1,336 @@
+----
+
+This new functionality is enabled by configuring either a bootstrap property, an environment variable, or through the `server.xml` file. If configuration is present in all these options, the configuration precedence is the `server.xml` > environment variable > bootstrap property.
+
+*Configuration:*
+
+- `bootstrap.properties`: 
+[source]
+----
+com.ibm.ws.logging.stackTraceSingleEntry=true
+----
+- `server.env`: 
+[source]
+----
+WLP_LOGGING_STACK_TRACE_SINGLE_ENTRY=true
+----
+- `server.xml`: 
+[source,xml]
+----
+<logging stackTraceSingleEntry="true" />
+----
+
+[#bugs]
+== Notable bugs fixed in this release
+
+
+We’ve spent some time fixing bugs. The following sections describe the issues resolved in this release. If you’re interested, here’s the  link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A22007+label%3A%22release+bug%22[full list of bugs fixed in 22.0.0.7].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/21441[The openapi-3.1 Liberty feature generates wrong property name for annotation @Schema]
++
+The `openapi-3.1` feature is used in Liberty to generate the Open API documents. We found that the `@Schema` annotation doesn't work as expected. 
+For example, when annotating a field as shown below:
++
+[source, java]
+----
+    @Schema(name="asset_id", readOnly = true, required = true)
+    private String assetId;
+----
++
+The generated swagger doc `.yaml` file will contain the following:
++
+[source, yaml]
+----
+RelatedAsset:
+  required:
+  - assetId         <<----- name of the property in the "required" section is not correct
+  - ...
+  type: object
+  properties:
+    asset_id:       <<----- name of the property in the "properties" section is correct
+      type: string
+      readOnly: true
+    ...
+----
++
+The property is generated correctly in the properties section however it was not generated correctly in the required section. The expected name should be the name used in the `@Schema` annotation, i.e., `asset_id`. This issue has now been resolved so that the correct property name for the `@Schema` annotation is generated.
```

Comment:
```suggestion
Although the property is generated correctly in the properties section, it was not generated correctly in the required section. The expected name should be the name used in the `@Schema` annotation, i.e., `asset_id`. This issue has now been resolved so that the correct property name for the `@Schema` annotation is generated.
```